Exiled.API/Features/Door.cs

```diff
@@ -78,7 +83,12 @@ public Door(DoorVariant door, Room room)
         /// <summary>
         /// Gets the <see cref="Features.Room"/> that the door is located in.
         /// </summary>
-        public Room Room { get; }
+        public Room Room => Rooms.Count == 0 ? null : Rooms.First();
```

Comment:
FirstOrDefault()

Reply:
Not needed, we do `Rooms.Count` check before that

Reply:
i mean instead of this Room.Count check